### ISSUE TITLE
Install fails with `--fast` flag on python 3.10.2

### ISSUE THREAD

**Issue by ljgray** [chime-experiment/mkchimeenv#5]
Install fails on Cedar when `--no-build-isolation` is set (which is made active by setting the `--fast` flag).

To reproduce:
1. Force purge modules and load `chime/python/2022.06`
2. Run `mkchimeenv create venv --fast`

Relevant traceback occurs at line 345 in `env.install`:
```
virtualenvapi.exceptions.PackageInstallationException: (1, b"Looking in links: 
/cvmfs/soft.computecanada.ca/custom/python/wheelhouse/gentoo2020/avx2, 
/cvmfs/soft.computecanada.ca/custom/python/wheelhouse/gentoo/avx2, 
/cvmfs/soft.computecanada.ca/custom/python/wheelhouse/gentoo2020/generic, 
/cvmfs/soft.computecanada.ca/custom/python/wheelhouse/gentoo/generic, 
/cvmfs/soft.computecanada.ca/custom/python/wheelhouse/generic\nObtaining 
file:///project/6003614/lgray/test_data/tmp/test/t2/code/caput\n  Checking if build backend supports build_editable: 
started\n  Checking if build backend supports build_editable: finished with status 'done'\n  Preparing metadata 
(pyproject.toml): started\n  Preparing metadata (pyproject.toml): finished with status 'error'\n", '-e 
/project/6003614/lgray/test_data/tmp/test/t2/code/caput')
```

Possibly related: https://github.com/pypa/pip/issues/6264

Install is successful if `--fast` flag is not set.

**Comment by ljgray** [comment]
Seems to be an issue with `versioneer` not getting picked up as a requirement. Most likely related to 53dcf74854c0c655e8c253679f66b2619e862257

**Comment by ljgray** [comment]
Fixed by https://github.com/radiocosmology/caput/pull/261

**Comment by ljgray** [comment]
This is due to versioner not being installed when `build isolation` in enabled

**Comment by ljgray** [comment]
Closed by #6

**Comment by ljgray** [comment]
Closed by #6
